pkg/kubelet/cm/types.go

```diff
@@ -90,8 +78,8 @@ type CgroupManager interface {
 	Pids(name CgroupName) []int
 	// ReduceCPULimits reduces the CPU CFS values to the minimum amount of shares.
 	ReduceCPULimits(cgroupName CgroupName) error
-	// GetResourceStats returns statistics of the specified cgroup as read from the cgroup fs.
-	GetResourceStats(name CgroupName) (*ResourceStats, error)
+	// MemoryUsage returns current memory usage of the specified cgroup, as read from the cgroupfs.
```

Comment:
Maybe specify that it is in bytes and not pages/kB/MB?

Reply:
Done. I have also removed that the value is read from cgroupfs as this is kind of obvious.

Reply:
This still reads `MemoryUsage returns current memory usage of the specified cgroup, as read from the cgroupfs.` tho.